components/Organization/ActivityOrganizationModal.tsx

```diff
@@ -0,0 +1,146 @@
+import { observer } from 'mobx-react';
+import { PureComponent, FormEvent, createRef } from 'react';
+import { Form, Button, ModalProps, Modal } from 'react-bootstrap';
+import { formToJSON } from 'web-utility';
+import {
+  Organization,
+  OrganizationModel,
+  OrganizationTypeName,
+} from '../../models/Organization';
+
+export interface OrganizationModalProps
+  extends Pick<ModalProps, 'show' | 'onHide'> {
+  store: OrganizationModel;
+  onSave?: () => any;
+}
+
+interface OrganizationForm extends Organization {
+  logoURI: string;
+}
+
+@observer
+export class OrganizationModal extends PureComponent<OrganizationModalProps> {
+  private form = createRef<HTMLFormElement>();
+
+  increaseId = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    const { currentTarget } = event,
+      { store, onSave } = this.props;
+    const { name, description, type, logoURI } =
+      formToJSON<
+        Pick<OrganizationForm, 'name' | 'description' | 'type' | 'logoURI'>
+      >(currentTarget);
+
+    await store.updateOne({
+      name,
+      description,
+      type,
+      logo: {
+        name,
+        description,
+        uri: logoURI,
+      },
+    });
+    onSave?.();
+    this.handleReset();
+  };
+
+  handleReset = () => {
+    this.form.current?.reset();
+    this.props.onHide?.();
+  };
+
+  render() {
+    const { show, onHide } = this.props;
+
+    return (
+      <Modal show={show} onHide={onHide} centered>
+        <Modal.Header closeButton>
+          <Modal.Title>增加主办方信息</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Form
+            ref={this.form}
+            onSubmit={this.increaseId}
+            onReset={this.handleReset}
+          >
+            <Form.Group className="mt-2">
+              <Form.Label htmlFor="name">名称</Form.Label>
+              <Form.Control
+                id="name"
+                name="name"
+                type="text"
+                placeholder="请输入名称"
+                required
+              />
+            </Form.Group>
+            <Form.Group className="mt-2">
+              <Form.Label htmlFor="description">描述</Form.Label>
+              <Form.Control
+                id="description"
+                name="description"
+                type="text"
+                placeholder="请输入描述"
+                required
+              />
+            </Form.Group>
+            <Form.Group className="mt-2">
+              <Form.Label htmlFor="type">类型</Form.Label>
+              <Form.Select
+                id="type"
+                name="type"
+                aria-label="Default select example"
+              >
+                {Object.entries(OrganizationTypeName).map(
+                  ([key, value], idx) => {
+                    return (
+                      <option key={idx + key} value={key}>
+                        {value}
+                      </option>
+                    );
+                  },
```

Comment:
```suggestion
                  ([key, value]) => (
                      <option key={key} value={key}>
                        {value}
                      </option>
                    ),
```
组件引擎渲染列表的 `key` 尽量与顺序无关